seleniumByTechnocredits/src/seleniumPractise/prachiJ/Assingment1/AlertDemo_Q1.java

```diff
@@ -0,0 +1,53 @@
+package seleniumPractise.prachiJ.Assingment1;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import util.Init;
+
+/*
+ * Alert Demo > Submit > Alert
+ * Validation of the values of Alert Demo with of the Alert.
+ */
+public class AlertDemo_Q1 {
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		String url = "http://automationbykrishna.com/#";
+		driver.get(url);
+		Thread.sleep(2000);
+		driver.findElement(By.xpath("//*[@class='nav navbar-nav navbar-right']/preceding-sibling::ul/li[4]/a")).click();
+		Thread.sleep(2000);	
+
+		AlertDemo_Q1 alertDemo_Q1 = new AlertDemo_Q1();
+		alertDemo_Q1.alertValuesValidation(driver);
+	}
+	
+	void alertValuesValidation(WebDriver driver) throws InterruptedException
+	{
+		String firstName = "Prachi";
+		String lastName = "Jain";
+		String compName = "CG";
+		WebElement e1 = driver.findElement(By.xpath("//input[@id='UserFirstName']"));
+		WebElement e2 = driver.findElement(By.xpath("//input[@id='UserLastName']"));
+		WebElement e3 = driver.findElement(By.xpath("//input[@id='UserCompanyName']"));	
+		e1.sendKeys(firstName);
+		e2.sendKeys(lastName);
+		e3.sendKeys(compName);
+		String fName = e1.getAttribute("value");
+		String lName = e2.getAttribute("value");
+		String cName = e3.getAttribute("value");
+		driver.findElement(By.xpath("//button[text()='Submit']")).click();
+		Alert alert = driver.switchTo().alert();
+		Thread.sleep(2000);
+		String alertMsg = alert.getText();
+		if((alertMsg.contains(fName)) && (alertMsg.contains(lName)) && (alertMsg.contains(cName)))
+		{
+			if(alert.getText().equals("Prachi and Jain and CG"))
```

Comment:
you have already do getText() on alert in alertMsg string then you can use that String, no need to again do alert.getText() in if statement
you can simply write if(alertMsg.equalsignorecase("Prachi and Jain and CG") 

also upper if statement is also not required. One if is sufficient.

Reply:
Ammended suggested changes